### DISCUSSION TITLE
Is there any way to group fields within a Bard set?

### DISCUSSION THREAD

**Discussion by ssang** [statamic/cms#5266] [Q&A]
For example, I have a media and content set and it'd be nice if I could have two tabs of fields, one for the content and one for the media. This would be much like how other blueprints work but I haven't found a way to get it to work within a Bard set.

**Comment by duncanmcclean** [comment]
You'd like the sets to be grouped in the 'set selector' dropdown? 

<img width="280" alt="image" src="https://user-images.githubusercontent.com/19637309/153953774-aafbb205-dcec-4c86-993b-3d1745c8dd92.png">

If so, no, it's not currently possible but would be a [decent feature request](https://github.com/statamic/ideas/issues/new)!

**Comment by ssang** [comment]
Not quite. For example, I believe if you create multiple sets of fields in a Blueprint for a Global, it will create a tabbed layout. I was hoping there would be a similar feature for fields within a Bard/Replicator Set.

Like my example above, if I have a Text and Media Block (Set) it'd be nice to have fields pertaining to the text aspect in one tab and fields for the media stuff in another tab. It's mostly so that a blocks list of fields doesn't end up becoming super long. 

I don't think it exists as I've even tried creating "sections" directly in the YAML file and it just gave me an error.
